ui/src/app/base/components/Section/_index.scss

```diff
@@ -10,6 +10,10 @@
     background-color: #fff;
   }
 
+  .section__header-icon {
```

Comment:
There's a util `.u-nudge-right--small` that would achieve the same thing if applied to the status text, so we might not need this class..

Reply:
Oh yeah, great, I was sure there was a util!